Resources/Locale/en-US/escape-menu/ui/options-menu.ftl

```diff
@@ -281,4 +281,3 @@ ui-options-enable-color-name = Add colors to character names
 ui-options-colorblind-friendly = Colorblind friendly mode
 ui-options-reduced-motion = Reduce motion of visual effects
 ui-options-chat-window-opacity = Chat window opacity
-ui-options-screen-shake-intensity = Screen shake intensity
```

Comment:
this is apart of the upstream locale file; not removing this doesn't effect anything aswell so you should keep it to avoid conflicts